artifactory.py

```diff
@@ -245,6 +246,19 @@ def chunks(data, size):
         yield {k: data[k] for k in islice(it, size)}
 
 
+def print_download_progress(offset, total_size, *args, **kwargs):
+    if total_size > 0:
+        msg = "Downloaded {}/{}MB...[{}%]".format(
+            int(offset / 1024 / 1024),
+            int(total_size / 1024 / 1024),
+            round(offset / total_size * 100, 2),
+        )
+    else:
+        msg = "Downloaded {}MB".format(int(offset / 1024 / 1024))
+
+    print(msg)
```

Comment:
Still print is here. Let's change to logging.debug

Reply:
I still believe that print function is more valid for users and not for debugging, thus I would ask to keep print

If some developer want to change the behaviour, they can suppress or write own function. I provided example for this in readme

Reply:
@allburov 
you know, I think I found an easy setup way. Let's just add another argument to print function.
I also added an example in readme